### PR TITLE
Added various actions and fixed health check.

### PR DESCRIPTION
**Pull request by cs-adarsh-gupta**
#### Descriptions:
Added various actions and fixed health check.

### CODE REVIEW

File: taegis-xdr/info.json [hunk 1]
```diff
@@ -582,193 +998,124 @@
...
       ]
     },
     {
-      "operation": "update_alert_status",
-      "title": "Update Alert Status",
-      "annotation": "update_alert_status",
-      "description": "Adds a resolution status or modifies an existing resolution status based on the alert IDs and other input parameters specified.",
+      "operation": "get_playbook_execution",
+      "title": "Get Playbook Execution",
+      "annotation": "get_investigations_alerts",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json]: Please check the annotation I believe its wrong

File: taegis-xdr/info.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "taegis-xdr",
   "label": "Taegis XDR",
-  "version": "1.0.0",
-  "description": "SecureWorks Taegis™ XDR offers superior detection, unmatched response and an open platform built from the ground up to integrate market-leading technologies and deliver the highest ROI.",
+  "version": "1.1.0",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json:4]: Arrange the actions as per the crud sequence

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: taegis-xdr/info.json (hunk 1 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json]: Resolved

File: taegis-xdr/info.json (hunk 2 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json:4]: Resolved

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**